packages/beeq/src/components/alert/bq-alert.tsx

```diff
@@ -0,0 +1,255 @@
+import { Component, Element, Event, EventEmitter, h, Host, Listen, Method, Prop, State, Watch } from '@stencil/core';
```

Comment:
the `Listen` can be removed, is no longer used in the component implementation 😉

Reply:
Ohh, yes, I forgot.